### ISSUE TITLE
docker-compose - Using private images

### ISSUE THREAD

**Issue by rgarbin** [testcontainers/testcontainers-java#346]
Hi,

How I could up my compose.yml using private images? 

I take a look and not found this option to notify testcontainers when up and run docker/compose

In my host I do docker login and trying my docker-compose from host, run ok, only by testcontainer the compose not work with private images.

```java
@ClassRule
public static DockerComposeContainer compose = new DockerComposeContainer(new File("src/test/resources/docker-compose-test.yml"));
```

**Comment by rgarbin** [comment]
Testcontainers use as docker client  the docker-java project, so, I only need a property file, right?

https://github.com/docker-java/docker-java

```bash
$ cd ~
$ vim .docker-java.properties
DOCKER_HOST=tcp://localhost:2376
DOCKER_TLS_VERIFY=1
DOCKER_CERT_PATH=/home/user/.docker/certs
DOCKER_CONFIG=/home/user/.docker
api.version=1.23
registry.url=https://index.docker.io/v1/
registry.username=myuser
registry.password=mypass
registry.email=mymail@gmail.com
```

**Comment by KORPSE** [comment]
Looks like it is connected with bug in Docker https://github.com/docker/compose/issues/4171
It was fixed in version 1.10, so I suggest to upgrade the version of docker-compose container in [here](https://github.com/testcontainers/testcontainers-java/blob/3b1bbebb47e953ef4660bec1f5e996580eb42604/core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java#L389)
Is it possible?

**Comment by kiview** [comment]
I don't think the issue is related, we could update the docker-compose image nevertheless though.
`DockerComposeContainer.withLocalCompose(true)` is a good workaround, what about mounting the `.docker/config.json` into the compose container? Should work as well.

**Comment by iNikem** [comment]
Isn't it fixed by #536 and can be closed now?

**Comment by kiview** [comment]
@iNikem is right, I'll close this issue.
Fixed by #537
